programs/ambassadors/ambassadors-program.md

```diff
@@ -0,0 +1,61 @@
+
+### What is the JSON Schema Ambassador Program?
+
+The JSON Schema Ambassador Program aims to bring JSON Schema closer to both current users and potential new project users. It also aims to provide users and community members with the necessary tools and resources to:
+
+- Promote collaboration among the JSON Schema ecosystem.
+- Engage with the project in some way, either as a contributor, blogger, speaker, etc.
+- Speak at community events or write technical content (e.g. blog posts) that solves community needs.
+- Nurture community growth and a welcoming environment to spread knowledge about the project.
+
+### How does the program work?
+
+- The ambassador selection process will be as transparent as possible.
+- The TSC and existing ambassadors will evaluate applications according to compliance requirements via a traditional PR flow.
+- All ambassadors information will be displayed on the JSON Schema website in a list after the PR is merged.
+
+### Ambassador requirements
+
+The participation will be determined and checked annually. The minimum requirement for being an ambassador is to make **4 contributions per year** in the form of articles, talks, videos, podcasts, presentations, driving initiatives, etc.
+
+Ambassadors can be invited to review content (i.e., articles, talks, videos) related to the community. These reviews are optional and will be counted as an additional contribution.
+
+Some clarifications on the type of contributions:
+    
+- Articles, videos, and podcasts can be published on [json-schema.org/blog](https://json-schema.org/blog) or other domains. 
+- Talks and presentations can be displayed at different events or workshops. Within them, JSON Schema should appear at the core of the presentation. The presentation should not be being used as a disguised marketing tool for other products/projects.
+- Special contributions should be driven to support the community or enhance its visibility (i.e., getting new sponsors or driving initiatives) 
+- If we do not notice or accidentally miscount a contribution, just let us know.
+         
+Examples: 
+
+1. If contributors publish 3 articles and propel 1 initiative about JSON Schema in eight months, they will become ambassadors for the whole year.
+2. If contributors make 1 presentation, write 2 articles, and propel 1 initiative, they will become ambassadors for the whole year.
+3. If you were accepted as ambassador on the 5th of May 2023, you have already fulfilled the requirements. On the 5th of May 2024, we will check if you qualify again after the 5th of May 2023.
+   
+### Ambassadors duties
+
+- Be in tune with JSON Schema's mission and values.
+- Always respect the [code of conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).
+- Be active in your role as an ambassador.
+
+### Ambassador benefits
+
+- Invitation to [the JSON Schema organization](https://github.com/orgs/json-schema-org/people).
```

Comment:
This is a bit concerning.  As far as I'm aware, members have GitHub write access like committing directly to repo branches (without making a fork), merging PRs, etc.  I'm not sure we want to extend that authority to people outside the core team / TSC.

Reply:
> This is a bit concerning.

I hear you and you are right. I'll replace this point by Visibility in the JSON Schema Ambassadors page of the JSON Schema website.

Reply:
It doesn't automatically. We can manage it on a team basis. We can have a team which has read only access to the public repos. It may give them permissions to crates issue labels for example, but I think even that can be limited.

Personally, I think if we can do it without giving permissions, we should. I've always liked having logos on my GitHub profile. There's a sense of pride in association.

Reply:
My concern is authority. If that can be managed, I'm okay with this.

Reply:
By default, any member or team member, provides no additional permissions, apart from creating a repo.
We could remove the permission for all members to create repos, which I think might actually be the right thing to do anyway. We don't need to create new repos often.